203.Remove_Linked_List_Elements.py

```diff
@@ -0,0 +1,19 @@
+# https://leetcode.com/problems/remove-linked-list-elements/
+
+def removeelements(head, val):
```

Comment:
Tried to solve it my way without prev node. The case with the head.val == val I check only once in the end.
if not head:
    return
cur = head

while cur.next:
    if cur.next.val == val:
        cur.next = cur.next.next
    else:
        cur = cur.next
if head.val == val:
    head = head.next
return head